components/Base/Button/Index.vue

```diff
@@ -11,6 +11,12 @@
       <slot name="icon" />
       <slot />
     </div>
+    <div v-if="loading" class="flex justify-between">
+      <span>
+        {{ textLoading }}
+      </span>
+      <jds-spinner size="14" />
+    </div>
```

Comment:
super minor:

if you just want to show a loading indicator, it's cheaper to use `v-show` instead of `v-if` because `v-show` only adding CSS instead of removing the element from the DOM

Reply:
Yes, I think is the right thing to use `v-show` in this case 🙏🏽